doc/reference/tooling/tt_cli/installation.rst

```diff
@@ -5,6 +5,11 @@ To install the ``tt`` command-line utility, use a package manager -- Yum or
 APT on Linux, or Homebrew on macOS. If you need a specific build, you can build
 ``tt`` from sources.
 
+.. NOTE::
+
+    A Tarantool Enterprise's `release package <https://www.tarantool.io/en/enterprise_doc/setup/#package-contents>`_ includes the `tt` utility extended with additional features like :ref:`importing <tt-crud-import>` and :ref:`exporting <tt-crud-export>` data.
```

Comment:
Maybe put this (or similar) text on the main **tt CLI** page to explain the existence/positioning of tt EE in general?

Reply:
Will add it to the intro